src/test/HostActivationTests/GivenThatICareAboutMultilevelSDKLookup.cs

```diff
@@ -2,15 +2,13 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
-using System.Threading;
 using Microsoft.DotNet.InternalAbstractions;
 using Xunit;
 
 namespace Microsoft.DotNet.CoreSetup.Test.HostActivation.MultilevelSDKLookup
 {
-    public class GivenThatICareAboutMultilevelSDKLookup
+    public class GivenThatICareAboutMultilevelSDKLookup : IDisposable
```

Comment:
Why do these tests have a different pattern than the other tests?  All the other tests use an `IClassFixture<SharedState>`, but this one doesn't.

Reply:
The classes that have global state (that used to be in statics) use IClassFixture, including SDKLookup and SharedFxLookup . The ones that used instance state just implement IDisposable.

However I do think some of the ones that use instance state could used shared state, but that would be an optimization that could come later.